homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -43,9 +44,20 @@ class LaCrosseSensorEntityDescription(
     """Description for LaCrosse View sensor."""
 
 
-def get_value(sensor: Sensor, field: str) -> float:
+def get_value(sensor: Sensor, field: str) -> float | int | str | None:
     """Get the value of a sensor field."""
-    return float(sensor.data[field]["values"][-1]["s"])
+    field_data = sensor.data.get(field)
+    if field_data is None:
+        LOGGER.warning(
+            "No field %s in response for %s (%s)", field, sensor.name, sensor.model
```

Comment:
What should the user do with this information?

Reply:
If an entity goes unavailable we should set the `available` property to return `False`.

Reply:
Setting available to false seems to me as the best way to go – the sensor is still configured in the app/cloud, so it should be listed, but it should report that no data for it is available. PR is here: https://github.com/home-assistant/core/pull/85303